ingestion/src/metadata/ingestion/source/atlas/atlas_lineage.py

```diff
@@ -0,0 +1,181 @@
+            yield from self.ingest_lineage(table)
+
+    def close(self):
+        pass
+
+    def get_status(self) -> SourceStatus:
+        return self.status
+
+    def _parse_table_entity(self, table_entity):
+        tbl_entities = table_entity["entities"]
+        for tbl_entity in tbl_entities:
+            try:
+                tbl_columns = self._parse_table_columns(table_entity, tbl_entity)
+                tbl_attrs = tbl_entity["attributes"]
+                db_entity = tbl_entity["relationshipAttributes"]["rdbms_db"]
+                db = self._get_database(db_entity["displayText"])
+                table_name = tbl_attrs["name"]
+                fqn = f"{self.config.service_name}.{db.name.__root__}.{table_name}"
+                tbl_description = (
+                    tbl_attrs["description"]
+                    if tbl_attrs["description"] is not None
+                    else " "
+                )
+
+                om_table_entity = Table(
+                    id=uuid.uuid4(),
+                    name=table_name,
+                    description=tbl_description,
+                    fullyQualifiedName=fqn,
+                    columns=tbl_columns,
+                )
+
+                table_and_db = OMetaDatabaseAndTable(table=om_table_entity, database=db)
+                yield table_and_db
+            except Exception as e:
+                logger.error(e)
+                logger.error(f"Failed to parse {table_entity}")
+                pass
+
+    def _parse_table_columns(self, table_response, tbl_entity) -> List[Column]:
+        om_cols = []
+        col_entities = tbl_entity["relationshipAttributes"]["rdbms_columns"]
+        referred_entities = table_response["referredEntities"]
+        dataset_name = tbl_entity["attributes"]["name"]
+        ordinal_pos = 1
+        for col in col_entities:
+            try:
+                col_guid = col["guid"]
+                col_ref_entity = referred_entities[col_guid]
+                column = col_ref_entity["attributes"]
+                data_type_display = tbl_entity["attributes"]["name"]
+                logger.info(column["data_type"])
+                logger.info(get_column_type(
+                    self.status, dataset_name, column["data_type"].upper()
+                ))
+                col_data_length = "1"
+                om_column = Column(
+                    name=column["name"],
+                    description=column.get("comment", None),
+                    dataType=get_column_type(
+                        self.status, dataset_name, column["data_type"].upper()
+                    ),
+                    dataTypeDisplay="{}({})".format(column["data_type"], "1"),
+                    dataLength=col_data_length,
+                    ordinalPosition=ordinal_pos,
+                )
+                om_cols.append(om_column)
+            except Exception as err:
+                logger.error(f"{err}")
+                continue
```

Comment:
This is the last line in the for loop and it will move to next iteration anyway

Remove continue?